testing/subreconciler.go

```diff
@@ -253,14 +255,19 @@ func (tc *SubReconcilerTestCase) Run(t *testing.T, scheme *runtime.Scheme, facto
 	}
 
 	// compare stashed
+	if tc.VerifyStashedValue == nil {
+		tc.VerifyStashedValue = func(t *testing.T, key reconcilers.StashKey, expected, actual interface{}) {
+			if diff := cmp.Diff(expected, actual, IgnoreLastTransitionTime, SafeDeployDiff, IgnoreTypeMeta, IgnoreCreationTimestamp, IgnoreResourceVersion, cmpopts.EquateEmpty()); diff != "" {
+				t.Errorf("Unexpected stash value %q (-expected, +actual): %s", key, diff)
+			}
+		}
+	}
```

Comment:
Move to before tc.Prepare is called. That way the Prepare method can wrap/replace the default behavior.